pjmedia/include/pjmedia/sdp.h

```diff
@@ -484,6 +484,17 @@ struct pjmedia_sdp_media
  */
 typedef struct pjmedia_sdp_media pjmedia_sdp_media;
 
+/**
+ * Print media description to a buffer.
+ *
+ * @param media	 The media description.
+ * @param buf	    The buffer.
+ * @param size	    The buffer length.
+ *
+ * @return	    the length printed, or -1 if the buffer is too
+ *		    short.
+ */
+PJ_DEF(int) pjmedia_sdp_media_print(const pjmedia_sdp_media *media, char *buf, pj_size_t size);
```

Comment:
- For declaration in header file, it should use `PJ_DECL` instead of `PJ_DEF`.
- The tabs/spaces in the param docs do not look properly aligned, but perhaps just the github diff preview.